src/Examples/UnitModels/Operations/statejunction_testing.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: StateJunction Unit Model with Modular Property Package\n",
+    "\n",
+    "![](statejunction.svg)\n",
+    "\n",
+    "## Learning Outcomes\n",
+    "\n",
+    "- Demonstrate use of the StateJunction unit model in IDAES\n",
+    "- Demonstrate different options available\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "In this example, we will be passing two different liquid streams through a StateJunction. The IDAES StateJunction unit model is a pass-through unit or simple pipe that is primarily used to link streams to/from different process alternatives in conceptual design applications. The inlet conditions are as follows:\n",
+    "\n",
+    "**Case 1:**\n",
```

Comment:
A test description of the stream would be nice here (i.e. a mixed stream of benzene and toluene).